airbyte-integrations/connectors/source-declarative-manifest/metadata.yaml

```diff
@@ -0,0 +1,27 @@
+data:
```

Comment:
Metadata in here could be sanity checked. it probably doesn't matter (release stage, ab_internal stuff for example). I generated a new definitionId.